src/main/scala/com/stratio/khermes/clients/http/protocols/WSProtocol.scala

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-package com.stratio.khermes.implicits
+package com.stratio.khermes.clients.http.protocols
 
-import org.json4s.{DefaultFormats, Formats}
+import com.stratio.khermes.clients.http.protocols.CommandsWSProtocol.CommandWsProtocol
 
-/**
- * Implicit values used in json4s to serialize / unserialize data.
- */
-trait KhermesSerializer {
+case class WSProtocol(command: CommandWsProtocol, args: Seq[String])
 
-  implicit val json4sFormats: Formats =  DefaultFormats
-}
+case object CommandsWSProtocol extends Enumeration {
```

Comment:
I have problems with sealed traits serializing / unserializing with json4s. Let me inspect deeper in the next iteration and I will change it.